apps/box/src/screens/Settings/ComponentGallery/ButtonGroupDemo.screen.tsx

```diff
@@ -0,0 +1,33 @@
+import {
+  FxBox,
+  FxButtonGroup,
+  FxSafeAreaBox,
+  FxSpacer,
+  FxText,
+} from '@functionland/component-library';
+import { HeaderText } from '../../../components/Text';
+import React from 'react';
+
+const ITEMS = ['first', 'second', 'third', 'fourth'];
```

Comment:
If you make this `as const` you can limit the idx type from being entered incorrectly

Reply:
Discussed offline